packages/gatsby-plugin-vtex-nginx/src/nginx-generator.ts

```diff
@@ -117,14 +117,21 @@ function generateRewrites(rewrites: Redirect[]): NginxDirective {
   }
 }
 
+function formatProxyHeaders(headers: Record<string, string> | undefined) {
```

Comment:
wouldn't that be equivalent?
```suggestion
function formatProxyHeaders(headers?: Record<string, string>) {
```

Reply:
The end behavior is the same, however the TypeScript compiler has different semantic interpretations of both forms. The second form allows us to use this function like so:
```
formatProxyHeaders()
```
whereas the if you try it with the first one, TypeScript compiler will fail with a `missing arguments` error message.

The first form is useful if you want to prevent the function to be called with no arguments, which, in this case, would mean an error of the developer. That's why I've chosen the first form